### PR TITLE
Add integration tests for GCS support in VariantWalkers

### PR DESCRIPTION
**Pull request by droazen**

### CODE REVIEW

**Comment by droazen** [comment]
@lbergelson please review

File: src/main/java/org/broadinstitute/hellbender/utils/test/BaseTest.java [hunk 1]
```diff
@@ -62,10 +62,10 @@ public void setTestVerbosity(){
     public static final String publicTestDir = new File(gatkDirectory, publicTestDirRelative).getAbsolutePath() + "/";
     public static final String publicTestDirRoot = publicTestDir.replace(publicTestDirRelative, "");
 
-    public static final String GSC_GATK_TEST_RESOURCES = "gs://hellbender/test/resources/";
+    public static final String GCS_GATK_TEST_RESOURCES = "gs://hellbender/test/resources/";
```
Comment (jean-philippe-martin) [inline src/main/java/org/broadinstitute/hellbender/utils/test/BaseTest.java]: Thanks for fixing the spelling. I'd suggest building those paths from command-line arguments, though, so we don't need to tell the whole world about this particular path.

**Comment by jean-philippe-martin** [comment]
Looks good to me otherwise!

File: src/main/java/org/broadinstitute/hellbender/utils/test/BaseTest.java (hunk 1 above)
Comment (lbergelson) [inline src/main/java/org/broadinstitute/hellbender/utils/test/BaseTest.java]: Oh, yeah, we have an environment variable for this so people can redirect to their own copy of the test resources if they want.  HELLBENDER_TEST_INPUTS.  We should use that instead.

Comment (lbergelson) [inline src/main/java/org/broadinstitute/hellbender/utils/test/BaseTest.java]: `Basetest.getGCPTestInputPath()` returns the appropriate test path.  (Maybe it should be changed to GCS?  I can't remember what gcp stood for?

Comment (droazen) [inline src/main/java/org/broadinstitute/hellbender/utils/test/BaseTest.java]: Done

**Pull request merged**